src/components/AudioPlayer/RepeatAudioModal/RepeatAudioModal.tsx

```diff
@@ -155,7 +155,9 @@ const RepeatAudioModal = ({
         </div>
       </Modal.Body>
       <Modal.Footer>
-        <Modal.Action onClick={onCancelClick}>Cancel</Modal.Action>
+        <Modal.Action onClick={isInRepeatMode ? onExitRepeatMode : onCancelClick}>
+          {isInRepeatMode ? 'Exit Repeat Mode' : 'Cancel'}
```

Comment:
Nice, two things: 

1. Let's change the copy to "Stop Repeating".
2. Make the color of stop repeating blue similar to the active repeat mode.